src/ProjectContents/Pagination.tsx

```diff
@@ -0,0 +1,91 @@
+import { BsArrowLeftShort, BsArrowRightShort } from "react-icons/bs";
+
+import * as css from "./Pagination.css";
+
+type PaginationProps = {
+  page: number;
+  pageLimit?: number;
+  contentsLimit?: number;
+  contentsCount: number;
+  onClickPage: (page: number) => void;
+};
+
+export function Pagination({
+  page,
+  pageLimit = 5,
+  contentsLimit = 20,
+  contentsCount,
+  onClickPage,
+}: PaginationProps) {
+  const totalPageCount = Math.ceil(contentsCount / contentsLimit);
+  const offset = (Math.ceil(page / pageLimit) - 1) * pageLimit;
+
+  const handleNextPageClick = () => {
+    page + pageLimit > totalPageCount
+      ? onClickPage(totalPageCount)
+      : onClickPage(page + pageLimit);
+  };
+
+  const handlePrevPageClick = () => {
+    page - pageLimit < 1 ? onClickPage(1) : onClickPage(page - pageLimit);
+  };
+
+  const handlePageNumberClick = (page: number) => {
+    onClickPage(page);
+  };
```

Comment:
```suggestion
  const handlePageNumberClick = onClickPage;
```

이렇게 줄여서 쓸 수 도 있어요..!